src/functions/template-tags/post.php

```diff
@@ -30,14 +30,18 @@ function tribe_get_the_content( $more_link_text = null, $strip_teaser = false, $
 		$content = get_the_content( $more_link_text, $strip_teaser );
 	}
 
-	/**
-	 * Filters the post content.
-	 *
-	 * @since 0.71 of WordPress
-	 *
-	 * @param string $content Content of the current post.
-	 */
-	$content = apply_filters( 'the_content', $content );
+	if ( ! doing_filter( 'the_content' ) ) {
+		/**
+		 * Filters the post content.
+		 *
+		 * @since 0.71 of WordPress
+		 *
+		 * @param string $content Content of the current post.
+		 */
+
+		$content = apply_filters( 'the_content', $content );
+	}
+
 	$content = str_replace( ']]>', ']]&gt;', $content );
```

Comment:
This could also go into the above conditional IMO

Reply:
This piece of code was added to prevent some mangling of the content in our filtering, I think shortcodes, from elsewhere. I think this should stay outside of the conditional.

Reply:
I suppose it doesn't really matter if we run it and then WP runs the same line again but just noted for reference